From d3e9f838c1e84abafab3541974194ecb6c5f758f Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 27 Apr 2016 17:48:37 +0200
Subject: [PATCH] Fix "pow" being undeclared
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

../video/out/opengl/lcms.c: In function ‘get_vid_profile’:
../video/out/opengl/lcms.c:235:27: error: implicit declaration of function ‘pow’ [-Werror=implicit-function-declaration]
             double binv = pow(src_black[i], 1.0/gamma);
                           ^
../video/out/opengl/lcms.c:235:27: warning: incompatible implicit declaration of built-in function ‘pow’
../video/out/opengl/lcms.c:235:27: note: include ‘<math.h>’ or provide a declaration of ‘pow’
---
 video/out/opengl/lcms.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/video/out/opengl/lcms.c b/video/out/opengl/lcms.c
index 7db8da6..a2030d3 100644
--- a/video/out/opengl/lcms.c
+++ b/video/out/opengl/lcms.c
@@ -16,6 +16,7 @@
  */
 
 #include <string.h>
+#include <math.h>
 
 #include "mpv_talloc.h"
 
-- 
2.7.4

